src/RedisResourceManager.php

```diff
@@ -233,9 +233,8 @@ protected function normalizeServer(&$server)
                 throw new Exception\InvalidArgumentException("Invalid server given");
             }
 
-            $host    = $server['host'];
-            $port    = isset($server['port']) ? (int) $server['port'] : $port;
-            $timeout = isset($server['timeout']) ? (int) $server['timeout'] : $timeout;
```

Comment:
This code block is not unused? 🤔

Reply:
🤷

Reply:
Why did you drop it then? :D

Reply:
It is actually unused as `$server['timeout']` is never available at this point. To get here `$server` must be a string that is then turned into array via `parse_url` containing no timeout key, or a hard `['host' => ...]` again without any timeout key.

Reply:
aha, I see: so `$server['timeout']` can never exist here 👍

Reply:
Exactly, sorry I misunderstood your first comment here.